Solution.yaml

```diff
@@ -808,3 +808,11 @@ runTemplates:
       - subdatasource
     parameterGroups:
       - etl_param_group_sub_dataset_by_dynamic_filter_with_multiple_selection
+  - id: filter1
```

Comment:
If this is a filter we want to keep, we should probably find a better id for it :smile:  (I don't want to make the same mistake as for our run templates, that are still named 1, 2 and 3 :sweat_smile:)